src/server/qgsserverprojectutils.cpp

```diff
@@ -195,6 +195,13 @@ bool QgsServerProjectUtils::wmsAddLegendGroupsLegendGraphic( const QgsProject &p
          || legendGroups.compare( QLatin1String( "true" ), Qt::CaseInsensitive ) == 0;
 }
 
+bool QgsServerProjectUtils::wmsSkipNameForGroup( const QgsProject &project )
+{
+  const QString skipNameForGroup = project.readEntry( QStringLiteral( "WMSSkipNameForGroup" ), QStringLiteral( "/" ), "" );
```

Comment:
Could you use the `readBoolEntry` method too in this case?

Reply:
Done